docker/php-fpm/Dockerfile

```diff
@@ -9,15 +9,12 @@ RUN apt-get update \
     && apt-get -y --no-install-recommends install php-xdebug php7.4-bcmath php7.4-gd php7.4-intl php-msgpack php-yaml \
     && apt-get clean; rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* /usr/share/doc/*
 
-# Install git
+# Install git and wget
 RUN apt-get update \
-    && apt-get -y install git \
+    && apt-get -y install git wget \
     && apt-get clean; rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* /usr/share/doc/*
 
-RUN mkdir -p /home/dev/.composer \
-    && chown -R 1000:1000 /home/dev/
+COPY --from=composer:2 /usr/bin/composer /usr/local/bin/composer
```

Comment:
This change inside this PR is correct or we should create another PR for this?
I don't know at the moment if is necessary for phar archive.

Reply:
correct is not necessary, but I was and I updated!  :)

Reply:
ok! :)